scalding-hadoop-test/src/test/scala/com/twitter/scalding/platform/PlatformTest.scala

```diff
@@ -437,11 +435,12 @@ object PlatformTest {
   }
 }
 
-class TestEmptySource extends FileSource with Mappable[(String, Long)] with SuccessFileSource {
+class TestEmptySource(val fields: Fields = new Fields("customNamedOffset", "customNamedLine")) extends FileSource with Mappable[(String, Long)] with SuccessFileSource {
```

Comment:
should we create a fields version of this source to decouple the fields and typed source?
e.g.
```
// use this in the existing tests
class TestEmptySource extends FileSource with Mappable[(String, Long)] with SuccessFileSource {
..
} 

// use in the new test
class TestFieldsEmptySource(val fields: Fields = new Fields("customNamedOffset", "customNamedLine")) extends FileSource with Mappable[(String, Long)] with SuccessFileSource {
...}
```